classes/classes/Scenes/Areas/GlacialRift/Amarok.as

```diff
@@ -0,0 +1,164 @@
+			}
+			else if (player.getEvasionRoll()) {
+				outputText("You jump out of the way before it can hit.", false);
+				combatRoundOver();
+				return;
+			}
+			else {
+				//Damageeee + stun! Reference to the legend of it slapping a kid with its tail, except minus the bone breaking.
+				outputText("The hit sends you stumbling back", false);
+				if (player.findPerk(PerkLib.Resolute) < 0 && rand(2) == 0) 
+				{
+					outputText(", stunning you.", false);
+					player.createStatusEffect(StatusEffects.Stunned, 0, 0, 0, 0);
+				}
+				else outputText(".", false);
+				var damage:int = ((str + 100) + rand(75));
+				damage = player.reduceDamage(damage);
+				player.takeDamage(damage, true);
+				if(player.HP <= 0) {
+					doNext(game.combat.endHpLoss);
+					return;
+				}
+			}
+			combatRoundOver();
+		}
+		
+		override public function defeated(hpVictory:Boolean):void
+		{
+			game.glacialRift.amarokScene.winAgainstAmarok();
+		}
+		
+		override public function won(hpVictory:Boolean, pcCameWorms:Boolean):void
+		{
+			if(pcCameWorms){
+				outputText("\n\nThe Amarok looks down at the worms you came, sniffs them, then snarls and walks away. It must consider you tainted meat.", false);
+				doNext(game.combat.cleanupAfterCombat);
+			} else {
+				game.glacialRift.amarokScene.amarokChowTime();
+			}
+		}
+		
+		public function Amarok() 
+		{
+			this.a = "the ";
+			this.short = "Amarok";
+			this.imageName = "amarok";
+			this.long = "You are fighting an Amarok, a massive wolf that seems set on hunting you. The buffeting snow does nothing to hide its thick, black fur, and hardly manages to even ruffle it. It has golden, hungry eyes that watch your every move and sharp teeth capable of crushing bones. It looms over you in both height and width, with well-defined muscles, long legs, and bulky paws with deadly claws only adding to its intimidating stature. Even its tail looks capable of knocking you down. It's the most normal animal-like creature you've seen here yet, a normal wolf despite its size, but is no less terrifying. You get the feeling this won't be an easy fight, especially considering it's not about to let you run away.";
+			this.createCock(8, 1.5, CockTypesEnum.WOLF);
+			this.balls = 2;
+			this.ballSize = 2;
+			this.cumMultiplier = 1;
+			this.createVagina(false, 1, 1);
+			this.gender = genderless;
+			this.pronoun1 = "it";
+			this.pronoun2 = "it";
+			this.pronoun3 = "its";
+			createBreastRow();
+			createBreastRow();
+			createBreastRow();
+			this.ass.analLooseness = ANAL_LOOSENESS_TIGHT;
+			this.ass.analWetness = ANAL_WETNESS_DRY;
+			this.tallness = 8*12;
+			this.hipRating = HIP_RATING_AVERAGE;
+			this.buttRating = BUTT_RATING_AVERAGE;
+			this.lowerBody = LOWER_BODY_TYPE_WOLF;
+			this.skinTone = "black";
+			this.skinType = SKIN_TYPE_FUR;
+			this.hairColor = "black";
+			this.faceType = FACE_WOLF;
+			this.earType = EARS_WOLF;
+			initStrTouSpeInte(110, 80, 85, 95);
```

Comment:
It's behaving like an animal because it _is_ an animal. It is a literal wolf. However, it's also a _deity wolf,_ meaning it's not dumb. And that's ignoring how wolves are highly intelligent creatures. As for allowing you to run from her pups, do you know, like, anything about wolves? Or animals in general? They have to leave their babies from time to time.  They can't watch them 24/7. I've done actual research for this NPC, yes it seems a little unrealistic but you know what else is unrealistic?? Growing 3 Coeurl dicks along with one demon and 2 tentacle dicks, having 3 breast rows, having cats ears, having 9 tails, the concept of demon sex cows, goblins, minotaurs, satyrs, being morphs, the entire fucking game is unrealistic. I think we can handle a wolf letting you escape from its pups after it tosses you at them for them to play with and eat and then leaving.

But if you're _really, **really**_ insistent on having sex with a goddamn 100% undeniable wolf, not even wolf-morph or biped, a literal wolf,

![viola](https://cloud.githubusercontent.com/assets/15647033/25490365/3cb97fee-2b32-11e7-847a-c9fdcd555d93.gif)

Have at it. I ain't writing it.